### PR TITLE
EOS-22429: Added Alex workflow

### PR DESCRIPTION
**Pull request by venkuppu-chn**
Signed-off-by: Venkatesh K <venkatesh.k@seagate.com>

## Problem Statement:
Use of Inclusive words are harmful such as gender favoring, polarizing, race related, religion inconsiderate, or other unequal phrasing in text.

## Solution Design:
Alex is the tool to scan the inclusive words and post the results in PR annotations

## Coding:

* [x] Did you follow coding standard and verified it with flake8? 

* [x] Did your address all Codacy issues?

## Testing:

* [ ] Unittest updated and all unittests are passing?

* [ ] Sanity/Self tests are updated (if applicable)?

* [x] Manual testing done covering happy path and non-happy path?

## Integration:

* [ ] If there is any interface change, did you communicate it to other components?
* [ ] If there is any interface change, other componenent gate gatekeepers are ready to accept the change?
* [ ] Did you complete deployment test if applicable?

## PR checklist:
* [x] Did you add Jira number to the PR?

* [x] DCO and cla-signed?

### CODE REVIEW

**Comment by jjxsg** [comment]
@venkuppu-chn Please add details about Alex workflow in PR template.

File: .github/workflows/alex_reviewdog.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```
Comment (jjxsg) [inline .github/workflows/alex_reviewdog.yml:2]: Should this be changed to 2021 now? @cdeshmukh

Comment (cdeshmukh) [reply .github/workflows/alex_reviewdog.yml:2]: Yes

**Review by cdeshmukh — COMMENTED** [review]
@venkuppu-chn please add details about what this workflow will enable

File: .github/workflows/alex_reviewdog.yml (hunk 1 above)
Comment (venkuppu-chn) [reply .github/workflows/alex_reviewdog.yml]: Done

**Review by jjxsg — APPROVED** [review]

**Review by cdeshmukh — APPROVED** [review]
Please rebase PR, its out of date

**Pull request merged**